src/common/src/config.rs

```diff
@@ -683,6 +683,11 @@ pub struct CacheConfig {
     #[serde(default)]
     #[config_doc(omitted)]
     pub meta_cache_eviction: CacheEvictionConfig,
+
+    /// Configure the capacity of the meta cache in MB explicitly for compactor.
+    // TODO(li0k): `compactor_meta_cache_capacity_mb` is only used for compactor meta cache configuration, it is not involved in the calculation of CN reserved memory, we consider removing it in the future..
```

Comment:
QQ: Does this field also benefit standalone deployment?

Reply:
I am thinking whether we should just not expose a config to overwrite the compactor meta cache capacity. Instead, we can just use `min(128MB, node memory * some fix ratio)` as the meta cache capactiy.

Reply:
Thanks, I took patrick's suggestion and used a calculated rule to determine the compactor's meta cache.
compactor_meta_cache_capacity_mb = min(128MB, avail_memory * 2%). And I restructured the configuration into Hummock/Compactor memory config so that standalone can also benefit.